.eslintrc.js

```diff
@@ -1,6 +1,114 @@
 module.exports = {
-    "extends": "airbnb-base",
-    rules:{
-        "linebreak-style": 0
-    }
-};
\ No newline at end of file
+	"extends": [
+		"airbnb",
+		"prettier",
+		"prettier/react"
+	],
+	"parser": "babel-eslint",
+	"parserOptions": {
+		"ecmaVersion": 8,
```

Comment:
In the `package.json`, we should additionally enforce the JS version with an [engines key](https://docs.npmjs.com/files/package.json#engines).  That way it's clear why we are parsing at version 8.